core/src/test/java/com/graphhopper/routing/util/parsers/BikeTagParserTest.java

```diff
@@ -267,6 +267,16 @@ public void testSpeedAndPriority() {
         way.setTag("highway", "primary");
         way.setTag("cycleway:bicycle", "designated");
         assertPriority(PREFER.getValue(), way);
+
+        way.clearTags();
+        way.setTag("highway", "path");
+        way.setTag("bicycle_road", "yes");
+        assertPriority(VERY_NICE.getValue(), way);
+
+        way.clearTags();
+        way.setTag("highway", "path");
+        way.setTag("cyclestreet", "yes");
+        assertPriority(VERY_NICE.getValue(), way);
```

Comment:
Please move these new tests and the above one for `cycleway:bicycle` into the method `testCycleway`. So that all tests with assertPriority are located on one place.